hack/e2e/values.venafi-kubernetes-agent.yaml

```diff
@@ -0,0 +1,12 @@
+metrics:
+  enabled: true
+config:
+  clusterName: venafi-kubernetes-agent-e2e
+  clusterDescription: |
+      A kind cluster used for testing the venafi-kubernetes-agent.
+  venafiConnection:
+    enabled: true
+crds:
+  enabled: true
+# Disable the default authentication
+authentication: null
```

Comment:
Michael's documentation doesn't mention disabling the old authentication mechanism,
but it is necessary, because otherwise the Pod won't be scheduled until the `agent-credentials` Secret is present.
Needs more thought.

Reply:
I've updated it to disable the mounted secret system if `config.venafiConnection.enabled=true`